src/main/java/htsjdk/tribble/gff/Gff3BaseData.java

```diff
@@ -9,32 +9,36 @@
     private static final String ID_ATTRIBUTE_KEY = "ID";
     private static final String NAME_ATTRIBUTE_KEY = "Name";
     private static final String ALIAS_ATTRIBUTE_KEY = "Alias";
-    final String contig;
-    final String source;
-    final String type;
-    final int start;
-    final int end;
-    final Strand strand;
-    final int phase;
-    final Map<String, String> attributes;
-    final String id;
-    final String name;
-    final String alias;
-
-    Gff3BaseData(final String contig, final String source, final String type,
-                 final int start, final int end, final Strand strand, final int phase,
+    private final String contig;
+    private final String source;
+    private final String type;
+    private final int start;
+    private final int end;
+    private final Double score;
```

Comment:
It looks like this is set to -1 as a sentinel for missing value.  That makes me thing this should be `double` so you don't have to worry about NPE anywhere.